Disable fail-fast for tests.

jameshilliard opened the pull request:
By default github cancels matrix tests if one fails, this behavior makes debugging test failures more difficult to isolate as it is not obvious if a failure is common to other tests if they get cancelled automatically.

davsclaus commented:
There is a merge conflict now. I wonder if you can take a look, and also maybe what other vertx repos are doing in this regard with fail-fast or not.

jameshilliard committed:
Disable fail-fast for tests.

By default github cancels matrix tests if one fails, this behavior
makes debugging test failures more difficult to isolate as it is
not obvious if a failure is common to other tests if they get
cancelled automatically.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>

jameshilliard commented:
rebased
